### DISCUSSION TITLE
(3/1 オンライン開催) オープンソースカンファレンス 2024 Online/Spring セミナ講演のお知らせ

### DISCUSSION THREAD

**Discussion by hori-takeshi** [toppers/users-forum#140] [Announcements]
TOPPERS プロジェクトでは、3/1(土), 2(日)に開催される「オープンソースカンファレンス 2024 Online/Spring」
　　https://register.ospn.jp/osc2024-online-spring/
で、次のセミナ講演を行う予定です。

2024-3-1 10:00~
オープンソースで作る箱庭ドローンシミュレーションの技術を徹底解説
講師: 森 崇（箱庭ラボ）
https://event.ospn.jp/osc2024-online-spring/session/1398262

参加希望の方は、
  https://ospn.connpass.com/event/303178/
から登録をお願い致します。
オンライン開催ですので、地方にお住いの方でも無理なく参加可能です。箱庭やドローンシミュレーションにご興味のある方は、ぜひご参加下さい。

**Comment by mitsut** [comment]
OSPNのYoutubeチャネルにセミナーの動画が公開されました
[オープンソースで作る箱庭ドローンシミュレーションの技術を徹底解説](https://www.youtube.com/watch?v=Kb6i3Q7-L8E)
